src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_params.py

```diff
@@ -32,6 +32,9 @@ def list_app_service_plan_names(prefix, action, parsed_args, **kwargs):#pylint:
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 existing_plan_name = CliArgumentType(overrides=name_arg_type, help='The name of the app service plan', completer=list_app_service_plan_names, id_part='name')
 existing_webapp_name = CliArgumentType(overrides=name_arg_type, help='The name of the web app', completer=list_webapp_names, id_part='name')
+sku_arg_type = CliArgumentType(type=str.upper,
+                               choices=['F1', 'FREE', 'D1', 'SHARED', 'B1', 'B2', 'B3', 'S1', 'S2', 'S3', 'P1', 'P2', 'P3'],
+                               help='The pricing tiers, e.g., F1(Free), D1(Shared), B1(Basic Small), B2(Basic Medium), B3(Basic Large), S1(Standard Small), P1(Premium Small), etc')
```

Comment:
Yes, there is a link "https://azure.microsoft.com/en-us/pricing/details/app-service/". I can add it if we like.
I didn't do it because those pages might get moved or deleted, and once that happens, cli will need to adjust the code and re-publish. Back in VS time, we maintained our own fwlinks for easier re-sync, but in cli we don't do that. 
To get it easier for more information, in the help I am using `pricing tier`, which is the same phrase portal uses, also the pricing page will be the top hit when you do web search on `azure web pricing tier`